.travis.yml

```diff
@@ -9,9 +9,9 @@ before_script:
 - npm prune
 script:
 - npm run build
-- if [ "$TRAVIS_PULL_REQUEST" = "false" ]; then npm run test:ci; fi
```

Comment:
I realise this is an "in-progress" PR so you may be adding this back in later but wanted to highlight the following - https://docs.travis-ci.com/user/pull-requests/#Pull-Requests-and-Security-Restrictions. I don't think makes any difference for this PR given that its `scottlogic/finput:tests-driver-refactoring` -> `scottlogic/finput:tests-driver-refactoring` (not a PR from an outside fork), but it will matter if this gets merged for PRs from other forks.

Reply:
Yes, the link you pointed is the reason why I am pushing directly to upstream :)